azurelinuxagent/ga/exthandlers.py

```diff
@@ -107,7 +107,8 @@ def parse_ext_status(ext_status, data):
     if substatus_list is None:
         return
     for substatus in substatus_list:
-        ext_status.substatusList.append(parse_ext_substatus(substatus))
+        if substatus is not None:
```

Comment:
@hglkrijger Would a better check be to see if `ext_status.substatuslist` is a list (that is, check for vs. check absence)?